utils/keywords.py

```diff
@@ -0,0 +1,53 @@
+import pke 
+def search_keywords(text):
+    """ Extracts keywords from the text using TextRank algorithm.
+    Args:
+        text (str): Text from which keywords are to be extracted.
+    Returns:
+        keyphrases (list): List of keyphrases extracted from the text.
+        keywords (list): List of keywords extracted from the text.
+        """
+    # define the set of valid Part-of-Speeches
+    pos = {'NOUN', 'PROPN', 'ADJ'}
+
+    # 1. create a TextRank extractor.
+    extractor = pke.unsupervised.TextRank()
+
+    # 2. load the content of the document.
+    extractor.load_document(input=text,
+                            language='en',
+                            normalization=None)
+
+    # 3. build the graph representation of the document and rank the words.
+    #    Keyphrase candidates are composed from the 33-percent
+    #    highest-ranked words.
+    extractor.candidate_weighting(window=2,
+                                pos=pos,
+                                top_percent=0.33)
+
+    # 4. get the 10-highest scored candidates as keyphrases
+    keyphrases = extractor.get_n_best(n=10)
+    keywords = [keyphrase for keyphrase, score in keyphrases]
+    return keyphrases,keywords
```

Comment:
```suggestion
    return [keyword for keyword, score in keyphrases]
```

Please return only keywords as the method name says. We don't care about the scores.

Reply:
Also you can change the 'search_keywords' to 'search_keyphrases', because in fact these are keyphrases - we get one or more words. And use this naming from now on.